python/singa/image_tool.py

```diff
@@ -384,7 +388,7 @@ def crop8(self, patch, num_case=1, inplace=True):
         if num_case == patch5 + patch3:
             count = patch5
         else:
-            sample_list = range(0, patch5 + patch3)
+            sample_list = list(range(0, patch5 + patch3))
```

Comment:
according to http://python-future.org/compatible_idioms.html
we can avoid adding a list for range by

    from builtins import range
    # or
    from past.builtins import range